lib/absinthe/lexer.ex

```diff
@@ -199,19 +199,17 @@ defmodule Absinthe.Lexer do
   #   - `"""` BlockStringCharacter* `"""`
   string_value =
     ignore(ascii_char([?"]))
-    |> traverse({:mark_string_start, []})
+    |> post_traverse({:mark_string_start, []})
     |> repeat_while(string_character, {:not_end_of_quote, []})
     |> ignore(ascii_char([?"]))
-    |> traverse({:string_value_token, []})
+    |> post_traverse({:string_value_token, []})
 
   block_string_value =
-    ignore(
-      string(~S("""))
-      |> traverse({:mark_block_string_start, []})
-    )
+    ignore(string(~S(""")))
+    |> post_traverse({:mark_block_string_start, []})
```

Comment:
Without this change of moving the `post_traverse` out of the `ignore` a "An unknown error occurred during parsing: key :token_location not found in: %{}" was seen in the tests. This was because `mark_block_string_start/5` was not called, and the compiler was complaining that it was not called. 

Changing this ensured that the `mark_block_string_start/5` was called.